pkg/storaged/utils.js

```diff
@@ -805,8 +832,9 @@
                 block,
                 usage: 'mounted',
                 location,
-                set_noauto: !is_top,
-                actions: is_top ? get_actions(_("unmount")) : [_("unmount")],
+                has_fstab_entry,
+                set_noauto: !is_top && !is_temporary,
+                actions: (is_top ? get_actions(_("unmount")) : [_("unmount")]).concat(has_fstab_entry ? [_("mount")] : []),
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19418-20231023-134501-b6ea4635-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)